### PR TITLE
update kubead-scope

### PR DESCRIPTION
**Pull request by my-git9**
/kind cleanup

update kubead-scope.md for the output of `keadm`

### CODE REVIEW

File: docs/proposals/keadm-scope.md [hunk 1]
```diff
@@ -134,32 +134,30 @@ In cloud node it shuts down the cloud processes of KubeEdge
...
+      --kube-config string   Use this key to set kube-config path, eg: $HOME/.kube/config (default "/root/.kube/config")
 
 ```
 
 ### keadm join --help
 
 ```
 
-"keadm join" command bootstraps KubeEdge's edge component.
-It checks if the pre-requisites are installed already,
```
Comment (wackxu) [inline docs/proposals/keadm-scope.md:159]: why remove these lines?

Comment (my-git9) [reply docs/proposals/keadm-scope.md:159]: image
This is the output for command `keadm join --help`，the output for code has been changed.

Comment (wackxu) [reply docs/proposals/keadm-scope.md:159]: ok

**Pull request closed without merging**